apps/website/versioned_docs/version-v2.x/use-cases/public-goods.md

```diff
@@ -1,54 +1,40 @@
-
-The quadratic formula used in this type of funding means that the matching funds increase exponentially with the number of individual contributors, which helps to amplify the impact of smaller donations and create a more equitable distribution of funds. This makes it possible for projects with a large number of small supporters to receive more funding than projects with just a few large supporters, which helps to level the playing field for smaller and less well-funded initiatives.
-
-Quadratic funding is made up of:
-
-1. a pool of funds to be distributed
-2. a voting system for distributing those funds
-
-Two common voting systems you may have used are 1-person-1-vote and 1-dollar-1-vote. Quadratic funding is a best-of-both-worlds solution where users vote with the **square root** of the money they spend — read on to learn more.
-
-# Problems with Existing Funding Mechanisms
-
-All common fund-distribution systems have pros and cons:
+Think crowdfunding and grant-giving where the community weighs the contribution of each project and decides how much of financial funding said project should receive.
 
-### Markets
+There are different flavors of funding mechanisms, you can find more information about each of them below:
 
-Markets can be thought of as fund-distributors: sellers are the fund recipients, and buyers are the voters deciding what will be funded. In this case, the "voting system" is 1-dollar-1-vote: each buyer has exactly as much potential influence over where funds go as they have money to spend. This is also the case for traditional crowdfunding platforms like Kickstarter.
+- [Simple Voting Funding](/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md)
+- [Quadratic Funding](/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md)
+- [Retroactive Public Goods Funding](/use-cases/public-goods-funding/retroactive-public-goods-funding/what-is-retroactive-public-goods-funding.md)
 
-An advantage of this system is that each voter's influence is directly proportional to their need (at least, if we ignore wealth inequality). The more Alice needs pizza, the more pizza she'll buy, and the more funding will go to pizzerias.
+## Simple Majority Funding
 
-The disadvantage is that rich voters have an outsize influence on what gets funded. Alice may need pizza exactly as much as Bob needs movie tickets, but if Bob is twice as wealthy as Alice, he has twice as many "votes" to spend on going to the movies.
+Public goods funding through a Simple Majority mechanism involves a pool of funds that is allocated based on the outcomes of a straightforward voting process. The percentage of votes received determines the corresponding percentage of the pool distributed.
```

Comment:
any links to actual resources?

Reply:
Reading more about simple majority voting, I think this is wrong. As I tried to explain what voting without QV was. Changing it to Simple Voting, which is based on % of votes = % of pool